### DISCUSSION TITLE
Running very slow

### DISCUSSION THREAD

**Discussion by samssara55** [shivamguys/irctc-cypress-automation#64] [General]
Automation Running very slow taking 1 minute to solving captcha and taking time to fillup passanger details. bro.. tatkal tickets becomes full on 1-2 minutes , how can we book tatkal using this?? but i appriciate ur work

**Comment by sidhant-khamankar** [comment]
Yes, the captcha text is harder to recognize during tatkal time than during normal time. EasyOCR predicts the wrong captcha several times. Prefer "MANUAL_CAPTCHA": true
